libraries/chain/CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ target_link_libraries( eosio_chain fc chainbase Logging IR WAST WASM Runtime
 target_include_directories( eosio_chain
                             PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" "${CMAKE_CURRENT_BINARY_DIR}/include"
                                    "${CMAKE_CURRENT_SOURCE_DIR}/../wasm-jit/Include"
-                                   "${CMAKE_CURRENT_SOURCE_DIR}/libraries/eos-vm/include"
+                                   "${CMAKE_CURRENT_SOURCE_DIR}/../eos-vm/include"
```

Comment:
To be clear: this doesn't look like something that should have mattered BSD vs Linux, so need to investigate the root issue.

Reply:
I reverted this change back I cannot reproduce build error (unable to find header file) any longer. This might be due to 2 reasons: initially this change was done first when build was failing and it was superseded by further makefile changes, build machine got additional 32gb (64 now) of RAM